deltachat-ios/Chat/Views/Reactions/ReactionsView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ReactionsView.swift
+//  deltachat-ios
+//
+//  Created by Nathan Mattes on 10.01.24.
+//  Copyright © 2024 merlinux GmbH. All rights reserved.
+//
+
+import UIKit
+import DcCore
+
+class ReactionsView: UIControl {
+
+    private let reactionsStackView: UIStackView
+
+    init() {
+        reactionsStackView = UIStackView()
+        reactionsStackView.axis = .horizontal
+        reactionsStackView.translatesAutoresizingMaskIntoConstraints = false
+        reactionsStackView.spacing = 2
+        reactionsStackView.isUserInteractionEnabled = false
+
+        super.init(frame: .zero)
+
+        addSubview(reactionsStackView)
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    private func setupConstraints() {
+        let constraints = [
+            reactionsStackView.topAnchor.constraint(equalTo: topAnchor),
+            reactionsStackView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            trailingAnchor.constraint(equalTo: reactionsStackView.trailingAnchor),
+            bottomAnchor.constraint(equalTo: reactionsStackView.bottomAnchor),
+        ]
+
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    public func configure(with reactions: DcReactions) {
+
+        let moreThanFiveReactions = reactions.reactions.count > 5
```

Comment:
empty line probably not on purpose here

```suggestion
    public func configure(with reactions: DcReactions) {
        let moreThanFiveReactions = reactions.reactions.count > 5
```